rviz_marker_tools/include/rviz_marker_tools/marker_creation.h

```diff
@@ -66,7 +66,8 @@ inline visualization_msgs::Marker& makeMesh(visualization_msgs::Marker& m, const
 }
 
 /// create an arrow along x-axis
-visualization_msgs::Marker& makeArrow(visualization_msgs::Marker& m, double scale = 1.0, bool tip_at_origin = false);
+visualization_msgs::Marker& makeArrow(visualization_msgs::Marker& m, double diameter, double length,
+                                      bool tip_at_origin = false);
```

Comment:
Please keep the old API and forward to the new function as `makeArrow(m, 0.1*scale, scale, tip_at_origin)`